### ISSUE TITLE
Advanced Essentia Storage has too high priority for golems

### ISSUE THREAD

**Issue by twothe** [Azanor/thaumcraft#1588]
When assigning a Golem to warded jars and an advanced Essentia storage, the Golem will prefer the advanced storage above all. However this storage only has a sucking strength of 24 and therefore should be preferred the least (which makes sense from a Essentia design perspective as well).

**Comment by MentalMouse** [comment]
Well, as far as design, the low suction means the reservoir can easily be drained by anything that's piped to it.  That's a problem for getting essentia into the reservoir with pipes, but not if you have golems doing input.

**Comment by justicesays** [comment]
Having tested this with a friends alchemy setup, it appears that the reservoir doesn't participate in the special behavior golems have for labelled/unlabelled/void jars. If you tag it, it simply acts as a target for the golems' other essentia task, moving essentia between two containers.
Noticed this when we attempted to use a reservoir as the input for a centrifuge cycle, the golems would just fill it and ignore all the empty labelled jars that could have taken the essentia.
In the end we just used a bunch of unlabelled input jars for the centrifuge cycle.
